### DISCUSSION TITLE
Enhancement: Use less expensive model on Google search plugin

### DISCUSSION THREAD

**Discussion by edsonmedina** [danny-avila/LibreChat#4023] [Troubleshooting] [answered]
### What features would you like to see added?

Currently the google search plugin is using gpt-4-turbo to transform results. This model has been superseeded by gpt-4o-mini which is much cheaper.

### More details

gpt-4-turbo
input: $10.00 / 1M tokens,  output: $30.00 / 1M tokens

gpt-4o-mini
input: $0.150 / 1M input tokens, output: $0.600 / 1M output tokens (66x cheaper input!!! and 5x cheaper output)

This would make the search much more affordable.

### Which components are impacted by your request?

Plugins

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
You need to set the agent model, best used with a preset for easier access
https://github.com/user-attachments/assets/a5e1a380-c7d6-4682-a762-b211dd638724

Replacing "Plugins" soon for a much better system with wider provider support: https://github.com/danny-avila/LibreChat/issues/3607
